nvOC

```diff
@@ -219,30 +219,23 @@ Restart()
 ############################### Start nvOC ####################################
 Start()
 {
-   if [ $LOCALorREMOTE == "LOCAL" ];then
-   pkill -f gnome-terminal
-   pkill -f 5watchdog
-   pkill -f 7telegram
-   pkill -f 6tempcontrol
-   pkill -e screen
-   pkill -f 8wtm
-    gnome-terminal --window-with-profile=mining
-    echo -e "${B}Miner Restart....${N}"; echo ""
-      fi
-
-   if [ $LOCALorREMOTE == "REMOTE" ];then
-        echo -e "${B}Stopping nvOC processes...Ghost${N}"; echo ""
-        # Kill off the nv_OC processes
    pkill -f gnome-terminal
    pkill -f 5watchdog
    pkill -f 7telegram
    pkill -f 6tempcontrol
    pkill -e screen
    pkill -f 8wtm
+   if [ $LOCALorREMOTE == "LOCAL" ]
+   then
+     gnome-terminal --window-with-profile=mining
+     echo -e "${B}Miner Restart....${N}"; echo ""
+   fi
+   if [ $LOCALorREMOTE == "REMOTE" ]
+   then
      export DISPLAY=:0
-    gnome-terminal --window-with-profile=mining
-    echo -e "${B}Miner Restart....${N}"; echo ""
-    fi
+     gnome-terminal --window-with-profile=mining
+     echo -e "${B}Miner Restart....${N}"; echo ""
```

Comment:
Move also those two lines outside, I think the only conditional command is the export in case of remote